packages/app/cypress/e2e/specs_list_e2e.cy.ts

```diff
@@ -237,6 +237,14 @@ describe('App: Spec List (E2E)', () => {
         cy.get('button').contains('23 Matches')
       })
 
+      it('normalizes directory path separators for Windows', function () {
```

Comment:
This will fail on `develop` for Windows right now because `e2e/accounts` won't yield any results on Windows. With my changes it passes